lib/tasks/address_lookup_items.rake

```diff
@@ -0,0 +1,355 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'csv'
+
+# Read in data from zipcode csv files.
+# Taken and optimized from the 'my_zipcode_gem' gem rake file.
+#
+namespace :pie4providers do
+  namespace :address_lookups do
+    DEFAULT_INPUT_DIR = Rails.root.join('db/lookup_data')
+    INPUT_FN_STATES = 'free_zipcode_data_all_us_states.csv'
+    INPUT_FN_COUNTIES = 'free_zipcode_data_all_us_counties.csv'
+    INPUT_FN_ZIPCODES = 'free_zipcode_data_all_us_zipcodes.csv'
+
+    DEFAULT_EXPORT_DIR = File.join(DEFAULT_INPUT_DIR, 'exports')
+
+    MIN_STATES_EXPECTED = 51
+    MIN_COUNTIES_EXPECTED = 3000
+
+    STATES_CSV_COLS = %i[abbr name].freeze
+    COUNTIES_CSV_COL_VALUES = %i[name state_id county_seat].freeze
+    COUNTIES_CSV_COL_HEADERS = %w[name state county_seat].freeze
+    ZIPCODES_CSV_COL_VALUES = %i[code city_id state_id county_id area_code lat lon].freeze
+    ZIPCODES_CSV_COL_HEADERS = %w[code city state county area_code lat lon].freeze
+    # -----------------------------------------------------------------
+
+    # import_all is about twice as fast
```

Comment:
Can we remove these comments now that benchmarking has been done and is recorded in the PR comments?

Reply:
oops -- yup. will do.